tests/unit/test_req_file.py

```diff
@@ -222,7 +222,37 @@ def test_yield_editable_requirement(self):
         filename = 'filename'
         comes_from = '-r %s (line %s)' % (filename, 1)
         req = install_req_from_editable(url, comes_from=comes_from)
-        assert repr(list(process_line(line, filename, 1))[0]) == repr(req)
+        actual, = process_line(line, filename, 1)
+        assert repr(actual) == repr(req)
+        assert actual.use_pep517 is None
+
+    @pytest.mark.parametrize('use_pep517, line, expected', [
+        # Test passing use_pep517=None.
+        (None, '-e git+https://url#egg=SomeProject', None),
+        (None, '--use-pep517 -e git+https://url#egg=SomeProject', True),
```

Comment:
Can we test with a non-editable, non-vcs requirements specifier as mentioned in the original issue?